MathProblems/670MaximumSwap.py

```
@@ -0,0 +1,32 @@
+         if num<=9:
+             return num 
+         num=list(str(num))
+         flag=False
+         for i in range(len(num)):
+             if flag:
+                 break
+             index=0
+             element=num[i]
+             for j in range(i+1,len(num)):
+                 if num[i]<num[j]:
+                    if element<=num[j]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)